sdk/textanalytics/Azure.AI.TextAnalytics/src/AbstractSummaryOperation.cs

```diff
@@ -297,5 +297,10 @@ async ValueTask<OperationState<AsyncPageable<AbstractSummaryResultCollection>>>
 
             return OperationState<AsyncPageable<AbstractSummaryResultCollection>>.Failure(rawResponse, requestFailedException);
         }
+
+        /// <summary>
+        /// To get the Id of the operation.
+        /// </summary>
+        public string GetOperationId() => Id;
```

Comment:
Text Analytics are also HLCs and already expose the operation ID via a property. Why duplicate that?